middleware/slash.go

```diff
@@ -60,7 +60,7 @@ func AddTrailingSlashWithConfig(config TrailingSlashConfig) echo.MiddlewareFunc
 
 				// Redirect
 				if config.RedirectCode != 0 {
-					return c.Redirect(config.RedirectCode, uri)
+					return c.Redirect(config.RedirectCode, trimMultipleSlashes(uri))
```

Comment:
Would a term like sanitzeUrl or normalizeUrl or somthing similar be more appropriate, which could be kept in the future, even if other steps then slash trimming are required.